.evergreen/config/test/_template-skip-crypt-shared.yml

```diff
@@ -0,0 +1,11 @@
+tasks:
+  - name: "test-skip_crypt_shared-%mongodbVersion%"
+    tags: [ "replicaset", "local", "%mongodbVersion%", "skip_crypt_shared" ]
+    commands:
+      - func: "bootstrap mongo-orchestration"
+        vars:
+          TOPOLOGY: "replica_set"
```

Comment:
Did you want to add auth here, so that replica set with auth was the standard test environment when we're only testing on a single topology? Or is that unnecessary?